### DISCUSSION TITLE
[Feature request] Multiple 'from' names

### DISCUSSION THREAD

**Discussion by kevjaro** [anonaddy/anonaddy#363] [Feature Requests]
It would be great if there was an option to have a different *from name* for each username. I haven't found this on the roadmap or anywhere else yet.

Right now, if I set my *from name* to **Username** and I send a mail from either `abc@PublicUsername.AnonAddy.com` or `abc@PrivateUsername.AnonAddy.com`, the recipient would see the mail coming from Username. 

I think it would be quite useful to have the option to set them individually, so that mails coming from `abc@PublicUsername.AnonAddy.com` would show as **PublicUsername**, and mails coming from `abc@PrivateUsername.AnonAddy.com` would show as **PrivateUsername**.

Reasoning: I don't need websites and companies knowing my full name when communicating via mail, but it would might make sense for friends, family or the workplace to be able to immediately see who they're communicating with.

Currently I've just set initials as a temporary measure but a better solution would be preferable.

**Comment by willbrowningme** [comment]
I agree, it's on my to do list to add this, thanks!

**Reply by roke-julian-lockhart** [reply]
@willbrowningme, per https://github.com/anonaddy/anonaddy/issues/171#event-7711897827, why is this now a discussion? I can't subscribe to solely closure of this now, because it's not an issue. Could you at least unlock the original issue, and somehow track this there too?

**Reply by willbrowningme** [reply]
There is an issue for this here - https://github.com/anonaddy/anonaddy/issues/279
